tests/test_main.py

```diff
@@ -113,3 +114,12 @@ def test_run_match_config_params() -> None:
         if key not in ("app_dir",)
     }
     assert config_params == run_params
+
+
+@pytest.mark.anyio
+async def test_run_invalid_host(caplog: pytest.LogCaptureFixture) -> None:
```

Comment:
```suggestion
async def test_exit_on_invalid_host() -> None:
```

Reply:
I added this based on your suggestion here, want me to remove still? https://github.com/encode/uvicorn/pull/1757#discussion_r1033017140

Reply:
Hold on... Then is the goal of the test really to test that uvicorn exits on an invalid host, or the goal was to "test that we send an error message on an invalid host"?

Reply:
I would be happy to keep it to test if Uvicorn exits properly (with intended error code). Uvicorn doesn't send a standard error messages at this moment. Different OS gives different error message.